File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("phantomjscloud_sdk")

describe("PhantomjscloudSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# Phantomjscloud SDK exists test

require "minitest/autorun"
require_relative "../Phantomjscloud_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = PhantomjscloudSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
